src/jit/importer.cpp

```diff
@@ -687,18 +687,30 @@ GenTreeStmt* Compiler::impExtractLastStmt()
     return stmt;
 }
 
-/*****************************************************************************
- *
- *  Insert the given GT_STMT "stmt" before GT_STMT "stmtBefore"
- */
-
+//-------------------------------------------------------------------------
+// impInsertStmtBefore: Insert the given GT_STMT "stmt" before GT_STMT "stmtBefore".
+//
+// Arguments:
+//    stmt       - a statement to insert;
+//    stmtBefore - an insertion point to insert "stmt" before.
+//
 inline void Compiler::impInsertStmtBefore(GenTreeStmt* stmt, GenTreeStmt* stmtBefore)
```

Comment:
I hope this function (and `impInsertTreeBefore` both have only one use) both have only one use each. I think they will go away soon so that is why they do not need more checks/cleaning right now.

Reply:
I trust you on that - though this is a bit confusing and I would have asked for a few more comments otherwise ;-)